app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="login-cont">
```

Comment:
- Kindly follow the design guidelines for the Login page:
    - [ ] Kindly add the `Back link` instead of the hamburger menu which redirects to the previous page.
    - [ ] Please add the `login link` in the navbar in the header responsible for the login process instead of adding a login button.
   - [ ] Kindly remove the input's label and use placeholders instead.
   - [ ] Kindly center the `Forget Password ?` link in the middle of the screen.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/Christelle-12/Budget-manager/assets/24703171/ee6edb6e-d7a1-4670-9739-97ac8bd574f2) | ![image](https://user-images.githubusercontent.com/24703171/219885103-63122090-4832-4d4f-b31f-0fc2e80b3777.png) |